samples/sensor/am2301b/app.overlay

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright (c) 2022 Hisanori Yamaoku
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+&i2c1 {
```

Comment:
Specifying an instance makes this overlay board-specific. I suggest changing this to `arduino_i2c` or moving the file to `samples/sensor/am2301b/boards/stm32_min_dev_blue.overlay`

Reply:
Moved to samples / sensor / am2301b / boards / stm32_min_dev_blue.overlay